### PR TITLE
TM-6124-Projected-Vacancies-UI

### PR DESCRIPTION
**Pull request by cagbakwu**
Dual Merge: 
- [API PR](https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/1015)

[Ticket](https://metaphase.atlassian.net/browse/TM-6124)

### CODE REVIEW

File: src/Components/ProjectedVacancyCard/ProjectedVacancyCard.jsx [hunk 1]
```diff
@@ -308,6 +309,7 @@ const ProjectedVacancyCard = (props) => {
             <PositionExpandableContent
               sections={sections}
               form={form}
+              tempHideEdit={isAO}
```
Comment (M00staff) [inline src/Components/ProjectedVacancyCard/ProjectedVacancyCard.jsx:312]: the ticket asked to hide the edit button - not to disable it, added this in for ya!

Comment (cagbakwu) [reply src/Components/ProjectedVacancyCard/ProjectedVacancyCard.jsx:312]: Good eye 👀

File: src/Constants/Menu.js [hunk 2]
```diff
@@ -469,6 +469,16 @@ export const GET_PROFILE_MENU = () => MenuConfig([
...
           ],
         } : null,
+      checkFlag('flags.projected_vacancy') ?
+        {
+          text: 'Projected Vacancy Management',
+          route: '/profile/ao/projectedvacancy',
+          icon: ' fa-keyboard-o',
+          roles: [
+            'superuser',
+            'bureau_user',
```
Comment (M00staff) [inline src/Constants/Menu.js:479]: i think this should have been `ao_user` 

... ill take care of it - tinkering with a few other things as well!

Comment (M00staff) [reply src/Constants/Menu.js:479]: surprised this worked for you locally?

there's a weird bug where you have to log out and back in twice locally to make sure your permissions come out correct.

Comment (cagbakwu) [reply src/Constants/Menu.js:479]: @M00staff I may have to clear my cache when reloading.

**Review by M00staff — APPROVED** [review]

**Pull request merged**